src/mmap.rs

```diff
@@ -708,7 +727,14 @@ mod tests {
             regions_summary
                 .iter()
                 .map(|(region_addr, region_size)| {
-                    Arc::new(GuestRegionMmap::from_range(*region_addr, *region_size, None).unwrap())
+                    Arc::new(
+                        GuestRegionMmap::from_range(&GuestMmapRange::new(
```

Comment:
```suggestion                  
GuestRegionMmap::from_range(&GuestMmapRange::new(
```

Reply:
Not sure I understand this one. The indentation is required here.